ext/SymbolicsLuxCoreExt.jl

```diff
@@ -0,0 +1,17 @@
+module SymbolicsLuxCoreExt
+
+using LuxCore, Symbolics
+
+@register_array_symbolic LuxCore.partial_apply(
+    model::LuxCore.AbstractExplicitContainerLayer, x::AbstractArray, ps::NamedTuple, st::NamedTuple) begin
+    size = ((model[end].out_dims),)
```

Comment:
Definitely won't work for anything other than `Dense`

Reply:
I've updated to `LuxCore.outputsize`.